<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rigidBodyesTab</class>
 <widget class="QWidget" name="rigidBodyesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="toolTabShortInfoBox" name="meshInfoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>191</width>
     <height>140</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="showRigidBodyRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show rigid bodies</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="rigidBodyesGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Rigid bodies list</string>
   </property>
   <widget class="QGroupBox" name="addGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Add new</string>
    </property>
    <widget class="QComboBox" name="meshSourceComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>44</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="rigidBodiesList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="massGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Mass</string>
   </property>
   <widget class="QDoubleSpinBox" name="massSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="massCenterGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Mass center</string>
    </property>
    <widget class="QDoubleSpinBox" name="massCenterXSpinBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X pos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y pos</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="massCenterYSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>43</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Z pos</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="massCenterZSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>66</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="geometricCenterButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Geometric center</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>toolTabShortInfoBox</class>
   <extends>QWidget</extends>
   <header location="global">widgets/tooltabshortinfobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
